package.json

```diff
@@ -157,6 +157,7 @@
     "tree-kill": "^1.0.0",
     "ts-node": "^1.3.0",
     "tslint": "^3.11.0",
-    "walk-sync": "^0.2.6"
+    "walk-sync": "^0.2.6",
+    "webpack": "^2.1.0-beta.23"
```

Comment:
The angular-cli creates a new project, and the repo itself is a dev-dep of the generated project. So in the scope of the `angular-cli`, webpack is a dependency, not a dev dependency.